docs/pages/kubernetes-access/guides/standalone-teleport.mdx

```diff
@@ -39,70 +47,188 @@ Use this command to switch to the cluster assigned to `CONTEXT_NAME`:
 $ kubectl config use-context ${CONTEXT_NAME?}
```

Comment:
I know it's not part of your changes, but curious... does the command above define `CONTEXT_NAME` or is it already set, or are we assuming but not instructing that the reader will set this in their current shell session?

Reply:
This assumed that the reader had assigned `CONTEXT_NAME`. I'll put a `Var` here!